src/Index.php

```diff
@@ -422,7 +423,9 @@ public function create(array $args = [], $options = null): Response
             }
         }
 
-        $endpoint = new Create();
+        unset($options['recreate']);
```

Comment:
Can you share why this is needed now?

Reply:
Yes, I had in mind to use option `wait_for_active_shards` when creating indexes on ES.
This would remove the need to have the function `Elastica\Test\Base::_waitForAllocation()` for example by simply passing this option when creating the index.

Reply:
I think I need to read up on my end what this flag does or doesn't. 

Is it now with the new code that `recreate` or was this always the case and now we remove it but didn't before? Trying to understand which part is related to the change of getting the options and what is further improvements.

Reply:
The unset is here only to remove invalid endpoint options so that these options can be passed to the endpoint via `setParams()`.